### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/etcher/commit/a5c5aaef0bbe9b6d7f047542acfbf46881699625#commitcomment-74541761

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/etcher/commit/20015d34f5cff287e142333751d5a6ed743a2365

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: .versionbot/CHANGELOG.yml [hunk 1]
```diff
@@ -184,7 +184,7 @@
   title: "'patch: updated electron to 12.2.3'"
   date: 2022-02-21T08:33:45.382Z
 - commits:
-    - subject: "patch: fix mesage of null"
+    - subject: "patch: fix message of null"
```
Comment (jsoref) [inline .versionbot/CHANGELOG.yml:187]: Some repositories don't like changing changelogs. I'm happy to drop these changes.

Comment (mcraa) [reply .versionbot/CHANGELOG.yml:187]: This is an auto-generated file from commit messages, should not be edited manually.

File: CHANGELOG.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 # v1.7.3
 ## (2021-12-29)
 
-* patch: fix mesage of null [Peter Makra]
+* patch: fix message of null [Peter Makra]
```
Comment (mcraa) [inline CHANGELOG.md:55]: This is a generated file aswell.

File: lib/gui/app/models/store.ts [hunk 3]
```diff
@@ -200,7 +200,7 @@ function storeReducer(
 							constraints.isDriveValid(drive, image) &&
 							!drive.isReadOnly &&
 							constraints.isDriveSizeRecommended(drive, image) &&
-							// We don't want to auto-select large drives execpt is autoSelectAllDrives is true
+							// We don't want to auto-select large drives except is autoSelectAllDrives is true
 							(!constraints.isDriveSizeLarge(drive) || shouldAutoselectAll) &&
```
Comment (lurch) [inline lib/gui/app/models/store.ts:204]: Looks like the `autoSelectAllDrives` variable might have been renamed to `shouldAutoselectAll` without the comment being updated to match? :man_shrugging:

Comment (jsoref) [reply lib/gui/app/models/store.ts:204]: Seems to just be a thing:
https://github.com/balena-io/etcher/blob/863999eab96dc17b8107e49595c1613ff1c077db/lib/gui/app/models/store.ts#L177-L178

File: lib/gui/app/models/store.ts [hunk 4]
```diff
@@ -200,7 +200,7 @@ function storeReducer(
 							constraints.isDriveValid(drive, image) &&
 							!drive.isReadOnly &&
 							constraints.isDriveSizeRecommended(drive, image) &&
-							// We don't want to auto-select large drives execpt is autoSelectAllDrives is true
+							// We don't want to auto-select large drives except is autoSelectAllDrives is true
```
Comment (lurch) [inline lib/gui/app/models/store.ts:203]: "except is" should probably be "except if" ?

Comment (jsoref) [reply lib/gui/app/models/store.ts:203]: I'll fix this this morning

**Comment by lurch** [comment]
"This branch has conflicts that must be resolved" :confused: 
I guess you need to rebase this on the latest version of `master` ?

**Comment by jsoref** [comment]
Done.

**Comment by aethernet** [comment]
@jsoref Can you add a `Change-type: patch` in the footer of one of the commit? 
Otherwise our CI will reject it.

**Comment by jsoref** [comment]
@aethernet done...

**Comment by dfunckt** [comment]
Thank you!

**Review by dfunckt — APPROVED** [review]

**Pull request merged**